.kokoro/firebase/quickstart-flutter.cfg

```diff
@@ -0,0 +1,8 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+build_file: "repository-gardener/.kokoro/build-flutter.sh"
+
+env_vars: {
+    key: "DPEBOT_REPO"
+    value: "firebase/quickstart-android"
```

Comment:
should this be quickstart-flutter?

Reply:
Yup, I am not as good at copying and pasting as I thought.

Updated!